cfme/tests/storage/test_volume.py

```diff
@@ -46,7 +51,48 @@ def volume(appliance, provider):
             msg=str(e)))
 
 
-def test_storage_volume_create_cancelled_validation(appliance, provider):
+@pytest.fixture(scope="function")
+def instance_fixture(appliance, provider, small_template):
+    instance = appliance.collections.cloud_instances.instantiate(random_vm_name('stor'),
```

Comment:
Optional:
If possible put full name "storage". It will help to debug if tear down fails.

Reply:
I wanted to, but random_vm_name has some functionality that it shortens the name to 4 letters anyway.